run_cart.sh

```diff
@@ -65,7 +66,8 @@ function run_bg {
 # run_server np cmd
 function run_fg {
         echo "run_fg CMD:"
-        cmd="orterun --mca btl tcp,self --mca oob tcp --mca mtl ^psm2,ofi --mca pml ob1 --map-by node --timeout $timeout --np $1 --hostfile $2 --output-filename testLogs/$4/sep_$test_$sep -x D_LOG_FILE=testLogs/$4/sep_$test_$sep/srv_output.log -x D_LOG_MASK=ERR -x CRT_PHY_ADDR_STR=\"ofi+verbs;ofi_rxm\" -x OFI_INTERFACE=$interface -x OFI_DOMAIN=mlx5_0  -x CRT_CTX_SHARE_ADDR=$sep -x CRT_CTX_NUM=16 -x PATH -x LD_LIBRARY_PATH $3"
+        let "sep++"
+        cmd="orterun --mca btl tcp,self --mca oob tcp --mca mtl ^psm2,ofi --mca pml ob1 --map-by node --timeout $timeout --np $1 --hostfile $2 --output-filename testLogs/$4/sep_$test_$sep -x D_LOG_FILE=testLogs/$4/sep_$test_$sep/srv_output.log -x D_LOG_MASK=ERR -x CRT_PHY_ADDR_STR=\"ofi+verbs;ofi_rxm\" -x OFI_INTERFACE=$interface -x OFI_DOMAIN=mlx5_0  -x CRT_CTX_SHARE_ADDR=$sep_cart -x CRT_CTX_NUM=16 -x PATH -x LD_LIBRARY_PATH $3"
```

Comment:
Same here for the mca params

Reply:
I recall there were few errors when you do not use some of the variables.